src/tools/rbd/action/Device.cc

```diff
@@ -177,6 +177,7 @@ void get_map_arguments(po::options_description *positional,
   options->add_options()
     ("show-cookie", po::bool_switch(), "show device cookie")
     ("cookie", po::value<std::string>(), "specify device cookie")
+    ("snap-id", po::value<std::string>(), "specify arbitrary snapshot id for mapping")
```

Comment:
Why is it added manually here instead of calling `at::add_snap_id_option` as done for unmap and detach?